### ISSUE TITLE
rhandsontable with shiny not working on mobile (touch) devices

### ISSUE THREAD

**Issue by bklingen** [jrowen/rhandsontable#96]
The app at https://artofstat.shinyapps.io/rhandsontableandshiny/ works fine, but when viewed on mobile devices (tablets, smartphones), the cells are not responsive and their content can't seem to be changed.(Sometimes I manage to input new values when tabbing several times on the cell, but this seems erratic.)

This is surprising as all your shiny examples on Git that are a year old, such as https://jrowen.shinyapps.io/rhandsontable_datafile/ work fine on mobile devices and one can change the cell contents.

Here is the ui.r and server.r of the first app above (modeled after the one in the example):

``` r
library(rhandsontable)
shinyUI(fluidPage(
  titlePanel("Edit Table"),
  rHandsontableOutput("hot")
))

shinyServer(function(input, output, session) {
  output$hot = renderRHandsontable({
    if (!is.null(input$hot)) {
      DF = hot_to_r(input$hot)
    } else {
      DF = data.frame(Var1=c("one","two"), 
                      Var2=3:4, 
                      Var3=c(TRUE,FALSE),
                      stringsAsFactors=FALSE)
    }
    rhandsontable(DF) %>% 
      hot_table(highlightCol = TRUE, highlightRow = TRUE)
  })
})
```

**Comment by jrowen** [comment]
It may just be coincidence that the previous apps worked on mobile, as [it doesn't look like handsontable currently support mobile.](http://docs.handsontable.com/0.26.1/demo-mobiles-and-tablets.html)

**Comment by bklingen** [comment]
Interesting, thanks.

**Comment by jackolney** [comment]
Just stumbled across this issue too. My shiny app displays tables rendered with rhandsontable but they cannot be edited. Annoyingly if you simulate an iOS size environment within Chrome you can edit cells, you just don't seem to be able to on the actual device.

Guessing this is a limitation of handsontable.js as opposed to the R package, bugger.

**Comment by SmarakNayak** [comment]
This issue looks like it was fixed in 0.28.1 [(#3750)](https://github.com/handsontable/handsontable/issues/3750). Any plans of updating the R package to incorporate the fix?

**Comment by jrowen** [comment]
`rhandsontable` is now using `v6.1.1` of `handsontable`, so this issue should be resolved.
